src/discussions/posts/post/Post.jsx

```diff
@@ -35,6 +36,7 @@ function Post({
   const { enableInContextSidebar } = useContext(DiscussionContext);
   const courseId = useSelector((state) => state.config.id);
   const topic = useSelector(selectTopic(post.topicId));
+  const CourseWideTopicURL = `${getConfig().BASE_URL}/${courseId}/topics/${post.topicId}`;
```

Comment:
rename to `courseWideTopicURL`

Reply:
if you are using this value one at one place when no need to create const. it would be better to directly use there.

Reply:
`CourseWideTopicURL` reflects to react component